index.html

```diff
@@ -1,12 +1,149 @@
 	<title>Bitcoin-Conference 2021 </title>
 </head>
+
 <body>
-	
+	<!-- Header section of the mobile website version starts here -->
+	<header>
+		<nav>
+			<button type="button" class="btn hambuger-menu"><svg xmlns="http://www.w3.org/2000/svg" width="30"
+					height="30" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
+					<path stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="M4 6h16M4 12h16M4 18h16" />
+				</svg></button>
+		</nav>
+		<section class="main-wrapper">
+			<div class="main-nav">
+				<ul id="main-bar">
+					<li><i class="fab fa-facebook-f"></i></li>
+					<li><i class="fab fa-twitter"></i></li>
+					<li><a href="#">Logout</a></li>
+					<li><a href="#">Mypage</a></li>
+					<li><a href="#">English</a></li>
+				</ul>
+			</div>
+			<div class="program-section">
+				<button class="nav__close" id="close-button" type="button">
+					<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor"
+						viewBox="0 0 24 24" stroke="currentColor">
+						<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+							d="M6 18L18 6M6 6l12 12" />
+					</svg>
+				</button>
+				<a href="index.html"> <img src="Assets/Icons/running.jpeg" id="logo" alt="logo"></a>
+				<ul class="nav-menu">
+					<li class="about"><a href="about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsors</a></li>
+					<li><a href="#">News</a></li>
+					<li id="camp"><a href="#">Campaign</a></li>
+				</ul>
+			</div>
+		</section>
+		<section class="home-page">
+			<p class="moto">Keeping You Healthy</p>
+			<h1 class="main-header">
+				RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT <br /> 2021 </h1>
+			<p class="main-text">Aqua's Gym is the Number one training facility in the west-side, A joyfull arena
+				where
+				we make you meet your fitness Goals happening yearly in over 30 states.</p>
+			<time datetime="2021-08-20"> 2021.08.20(FRI) &sim; 26(THU) </time>
+			<blockquote>&commat; Mana Convention Center Wynwood Miami</blockquote>
+		</section>
+	</header>
+	<section class="main-program">
+		<h2>Main Program</h2>
+		<div class="event-container">
+			<article class="main-event">
+				<img src="Assets/Icons/lecture.png" alt="lecture-flaticon">
+				<h3>Lecture</h3>
+				<p class="lectures">Listen to top fitness coaches and experts as they share their knowledge on how to
+					live a healthy
+					lifestle.</p>
+			</article>
+			<article class="main-event">
+				<img src="Assets/Icons/exhibiiton.png" alt="exhibition-flaticon">
+				<h3>Exhibition</h3>
+				<p class="workshop">Take a look around and get inspired by the different ways you can keep yourself fit
+					and healthy.
+				</p>
+			</article>
+			<article class="main-event">
+				<img src="Assets/Icons/forum.png" alt="forum-flaticon">
+				<h3>Forum</h3>
+				<p class="forum">Have the opportunity to share your thoughts and opinions with experts on topics that
```

Comment:
- [ ] Kindly reduce the spacings between each article in order to be in sync with the provided template.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/125090206-f58e1880-e083-11eb-8335-4617fe8eec26.png) | ![image](https://user-images.githubusercontent.com/49621279/125090253-00e14400-e084-11eb-94ec-718ef11d703f.png)
  
  - [ ] This section is distorted on tablet screens(768 px)
  
  _See image below_
    
![image](https://user-images.githubusercontent.com/49621279/125091591-6550d300-e085-11eb-845b-2e0b9814a2e4.png)

Reply:
I would attach a picture of this section from my machine, as it's not distorted, I have three screens, and they all work perfectly well. 
![Screen Shot 2021-07-09 at 3 37 02 PM (2)](https://user-images.githubusercontent.com/54026531/125095173-d0bd9380-e0cb-11eb-943d-2d08b14b2321.png)
![Screen Shot 2021-07-09 at 3 37 02 PM](https://user-images.githubusercontent.com/54026531/125096684-4b3ae300-e0cd-11eb-8b70-a0432d29dfd3.png)

here are two picture samples, one from a 32 inches Samsung monitor at 768 screen size.
the second is from a MacBook 15.5 inches at 768 screen size. please tell me where you are looking at this from or from which machine you are using. maybe a live video link of the full project would make it more clearer for you?

Reply:
![Screen Shot 2021-07-09 at 4 04 12 PM](https://user-images.githubusercontent.com/54026531/125099008-7a525400-e0cf-11eb-8894-66392f25a8a2.png)
![Screen Shot 2021-07-09 at 4 03 33 PM (2)](https://user-images.githubusercontent.com/54026531/125099060-8807d980-e0cf-11eb-927e-1f3cd4cf9d8a.png)


are other two images of different screens at 100% width. from a MacBook air of 13.3 inches the other my second Samsung screen.  please take a look at this pictures

Reply:
Hi @Geoelycom, since this is a matter of differences in screen sizes, I have marked it as optional. 🙂

Reply:
> Hi @Geoelycom, since this is a matter of differences in screen sizes, I have marked it as optional. 🙂

I have already submitted the project again for a review, which would make it my last I believe.